tutorial/gui/pcl-visualizer/ClassUsingPclVisualizer.cpp

```diff
@@ -0,0 +1,204 @@
+//! \example ClassUsingPclVisualizer.cpp
+#include "ClassUsingPclVisualizer.h"
+
+#if defined(VISP_HAVE_PCL)
+// PCL
+#include <pcl/io/pcd_io.h>
+
+// Visp
+#include <visp3/core/vpTime.h>
+#include <visp3/core/vpGaussRand.h>
+#include <visp3/core/vpRobust.h>
+#include <visp3/gui/vpColorBlindFriendlyPalette.h>
+#include <visp3/io/vpKeyboard.h>
+
+//! [Z coordinates computation]
+double zFunction(const double &x, const double &y, const unsigned int order)
+{
+  const double offset(0.5);
+  double z(0.);
+
+  for(unsigned int n = 0; n <= order; n++)
+  {
+    for(unsigned int k = 0; k <= order - n; k++)
+    {
+      if(k + n > 0)
+      {
+        z += std::pow(x, n) * std::pow(y, k);
+      }
+      else
+      {
+        z += offset;
+      }
+
+    }
+  }
+
+  return z;
+}
+//! [Z coordinates computation]
+
+//! [Constructor]
+ClassUsingPclVisualizer::ClassUsingPclVisualizer(std::pair<double, double> xlimits, std::pair<double, double> ylimits, std::pair<unsigned int, unsigned int> nbPoints)
+  : _t(0.1,0.1,0.1)
+  , _R(M_PI_4,M_PI_4,M_PI_4)
+  , _cMo(_t,_R)
+  , _minX(xlimits.first)
+  , _maxX(xlimits.second)
+  , _n(nbPoints.first)
+  , _minY(ylimits.first)
+  , _maxY(ylimits.second)
+  , _m(nbPoints.second)
+  , _visualizer("Grid of points with / without robust")
```

Comment:
Prefix vars with `m_`

Reply:
Done in [27c70a4](https://github.com/lagadic/visp/pull/1163/commits/27c70a448072c74aa1c7645df06011ec2de0faaf)